Backpack/src/main/res/values/attrs.xml

```diff
@@ -39,6 +39,19 @@
       <enum name="hero3" value="10" />
       <enum name="hero4" value="11" />
       <enum name="hero5" value="12" />
+      <enum name="heading1" value="13" />
```

Comment:
that's the new styles - unfortunately we can't deprecate the old ones in xml as android studio doesn't actually support it :(